src/operator/nn/dropout.cc

```diff
@@ -119,18 +119,12 @@ Example::
   out_shape->clear();
   out_shape->push_back(dshape);
   if (param.axes.ndim() > 0) {
-    // TODO (lnyuan): support specifying axes
-    LOG(FATAL) << "not supported yet";
-    /*
     for (int i = 0; i < param.axes.ndim(); ++i) {
       dshape[param.axes[i]] = 1;
     }
-    out_shape->push_back(dshape); */
-  } else {
-    mxnet::TShape mshape(1, static_cast<dim_t>(ceil(static_cast<double>(dshape.Size()) / 8)));
-    out_shape->push_back(mshape);
   }
-
+  mxnet::TShape mshape(1, static_cast<dim_t>(ceil(static_cast<double>(dshape.Size()) / 8)));
```

Comment:
mxnet::TShape mshape(1, static_cast<dim_t>((dshape.Size() + 7) / 8));

Reply:
because it's faster?

Reply:
Yes : )